curriculum/challenges/english/10-coding-interview-prep/rosetta-code/balanced-brackets.md

```diff
@@ -8,7 +8,7 @@ dashedName: balanced-brackets
 
 # --description--
 
-Determine whether a generated string of brackets is balanced; that is, whether it consists entirely of pairs of opening/closing brackets (in that order), none of which mis-nest.
```

Comment:
This particular change intrigued me because I've never seen the word "misnest", my browser's spellchecker doesn't recognize it. When seeing "mis-nest" I immediately knew what it meant.

I went to the internet for help, but I wasn't able to find any firm guidance on the subject of hyphenated prefixes. The most comprehensive and potentially-authoritative guidance I found was [from the APA style blog](https://blog.apastyle.org/apastyle/2016/10/hyphenation-station-the-hyphenation-of-prefixes-in-apa-style.html) and it really doesn't provide much help for this situation (one where we're basically inventing a word).

Probably the right solution would be to abandon the made-up word. Maybe something like, "that is, whether all opening brackets are followed by a closing bracket of the same type at the same level of nesting."